staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -275,6 +275,9 @@ func (o *LogsOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, args []str
 			return errors.New("expected a resource")
 		}
 		o.Object = infos[0].Object
+		if o.Selector != "" && len(o.Object.(*corev1.PodList).Items) == 0 {
+			return fmt.Errorf("no resources found in %s namespace", o.Namespace)
```

Comment:
It's not an error that you didn't find anything, let's just print this error similarly how `kubectl get` does, see here: https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/pkg/cmd/get/get.go#L585

Reply:
@soltysh 
Thanks, I have changed it now.